CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ Here is an overview of all new **experimental** features:
 ### Improvements
 
 - **General**: Add OPENTELEMETRY flag in e2e test YAML ([#5375](https://github.com/kedacore/keda/issues/5375))
+- **General**: Migrate away from unmaintained golang/mock and use uber/gomock ([#5440](https://github.com/kedacore/keda/issues/5440))
```

Comment:
Should we move this to `Others` section ? Wdyt @zroubalik

Reply:
Yes!

Reply:
done, PTAL!